openfisca_france/parameters/prestations/minima_sociaux/aah/montant.yaml

```
@@ -49,13 +49,13 @@ values:
   2017-04-01:
     value: 810.89
   2018-04-01:
-    value: 819
+    value: 819.00
   2018-11-01:
-    value: 860
+    value: 860.00
     reference: https://www.legifrance.gouv.fr/eli/decret/2018/10/31/SSAA1822428D/jo/article_1
   2019-11-01:
-    value: 900
+    value: 900.00
     reference: https://www.service-public.fr/particuliers/vosdroits/F12242
   2020-04-01:
-    value: 902.7
-    reference: https://www.legifrance.gouv.fr/loda/id/JORFTEXT000041835211/2020-10-13/
+    value: 902.70
+    reference: https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006073189&idArticle=LEGIARTI000020037365&dateTexte=29990101&categorieLien=cid
```

```suggestion
    reference: https://www.legifrance.gouv.fr/loda/id/JORFTEXT000041835211/2020-10-13/
```

---

Je garderais la référence d'origine. Es-tu ok @guillet ?

---

Effectivement, à l'emploi du [lien proposé](https://www.legifrance.gouv.fr/affichCodeArticle.do?cidTexte=LEGITEXT000006073189&idArticle=LEGIARTI000020037365&dateTexte=29990101&categorieLien=cid) par @mtifarine, le site `legifrance` redirige vers [ce lien de 2016](https://www.legifrance.gouv.fr/codes/id/LEGIARTI000031781089/2016-01-01/) qui n'est pas adapté à la revalorisation 2020 du montant d'AAH.

L'ancien lien dont tu parles @benjello est plus juste ; voire, celui-ci qui mène directement à l'article : 
```suggestion
    reference: https://www.legifrance.gouv.fr/jorf/article_jo/JORFARTI000041835215
```